src/settings/Settings.cpp

```diff
@@ -513,6 +513,8 @@ EZSeamType Settings::get<EZSeamType>(const std::string& key) const
         return EZSeamType::SHARPEST_CORNER;
     case "plugin"_sw:
         return EZSeamType::PLUGIN;
+    case "support"_sw:
+        return EZSeamType::SUPPORT;
```

Comment:
`SUPPORT` is not seam strategy. I wouldn't add it like this. Support should add a condition where the seam is not allowed to be placed. This should be combined with a steam stragegy but is not a stream strategy in it's own right.

Having disallowed areas for seam placement will be very handy in the future aswell for features like "no support in overhangs" etc.

Reply:
removed SUPPORT type z sea here: https://github.com/Ultimaker/CuraEngine/pull/2067/commits/70e33aa13c2d81b109b6bee8a4d898f4d1659724